packages/pangea-sdk/pangea/services/authn/models.py

```diff
@@ -434,281 +287,205 @@ class ClientJWKSResult(PangeaResponseResult):
     keys: List[Union[JWKec, JWKrsa, JWK]]
 
 
-# - path: authn::/v1/flow/complete
-# https://pangea.cloud/docs/api/authn#complete-sign-up-in
-class FlowCompleteRequest(APIRequestModel):
-    flow_id: str
-
-
-class FlowCompleteResult(PangeaResponseResult):
-    refresh_token: LoginToken
-    login_token: LoginToken
-
-
-# - path: authn::/v1/flow/enroll/mfa/complete
-# https://pangea.cloud/docs/api/authn#complete-mfa-enrollment
-class FlowEnrollMFACompleteRequest(APIRequestModel):
-    flow_id: str
-    code: str
-    cancel: Optional[bool] = None
-
-
-class EnrollMFAStart:
-    mfa_providers: List[str]
-
-
-class TOTPsecret:
-    qr_image: str
-    secret: str
-
-
-class EnrollMFAComplete:
-    totp_secret: TOTPsecret
+class UserAuthenticatorsDeleteRequest(APIRequestModel):
+    user_id: str
+    mfa_provider: MFAProvider
```

Comment:
the user authenticator delete request takes in an authenticator type, not an mfa provider.